app/components/UI/Stake/components/EstimatedAnnualRewardsCard.tsx

```diff
@@ -59,7 +59,10 @@ const EstimatedAnnualRewardsCard = ({
           <Text variant={TextVariant.BodyMDMedium}>
             {strings('stake.metamask_pool')}
           </Text>
-          <TouchableOpacity onPress={onIconPress}>
+          <TouchableOpacity
+            onPress={onIconPress}
+            accessibilityLabel="Learn More Icon"
```

Comment:
S: Adding "Icon" is unnecessary unless it's critical to explain that it's an icon and not a text or button.
```suggestion
            accessibilityLabel="Learn More"
```